utils/file-logging/src/lib.rs

```diff
@@ -95,12 +167,26 @@ pub fn start_logger(
 
     let log_spec = log_spec_builder.finalize();
     let mut logger = Logger::with(log_spec).format(log_format);
+
+    let mut messages = Vec::new();
     if let Some(log_dir) = log_dir {
-        logger = set_logging_to_files(logger, log_dir);
+        let config = FileLogConfig::with_env(log_dir, &mut messages);
+        messages.push((
+            log::Level::Info,
+            format!("Logging configuration: {config:#?}"),
+        ));
```

Comment:
It looks a little bit over-engineered

Reply:
I don't see any other way to properly report the error here. We could just abort the program, but I think logging errors is more in-line with the design of yagna. Of course you can't just log directly, because the logger can't be initialized here.